docs/features/diff-viewer/diff-viewer-rest-api.md

```diff
@@ -0,0 +1,98 @@
+# Diff Viewer REST API
+
+The Diff Viewer REST API is intended for people who want to include visual regression testing in their build pipeline. E.g. you could make your build fail if the overall change rate is beyond a certain threshold.
+
+Parameters:
+
+* `branchName` / `buildName` in URL: build to compare with the comparison build
```

Comment:
I would mention that this is the build where the comparison will be claculated on (where you can finally view it)